src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -25,6 +25,7 @@
 
 .card
   .card-body.p-0
+<<<<<<< HEAD
```

Comment:
@rubhanazeem seems some left over line from rebasing

Reply:
I see, present in one commit and removed in another one. Maybe you can remove it in the commit where it got overlooked. There is also a `binding.pry` added in one commit, and removed in another one.

Reply:
I'll split the PR into smaller chunks.